apps/system-e2e/src/tests/islandis/service-portal/smoke/health.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { test, BrowserContext, expect } from '@playwright/test'
+import { session } from '../../../../support/session'
+import { label } from '../../../../support/i18n'
+import { urls } from '../../../../support/urls'
+import { messages } from '@island.is/service-portal/health'
+import { disableI18n } from '../../../../support/disablers'
+
+test.use({ baseURL: urls.islandisBaseUrl })
+test.describe('Health pages', () => {
+  let context: BrowserContext
+
+  test.beforeAll(async ({ browser }) => {
+    context = await session({
+      browser: browser,
+      storageState: 'service-portal-evropa.json',
+      homeUrl: `${urls.islandisBaseUrl}/minarsidur`,
+      phoneNumber: '0102719',
+      idsLoginOn: true,
+    })
+  })
+
+  test.afterAll(async () => {
+    await context.close()
+  })
+
+  test('Overview page', async () => {
+    const page = await context.newPage()
+    await disableI18n(page)
+
+    await test.step('Checking for name and see more', async () => {
+      await page.goto('/minarsidur/heilsa')
+
+      const name = page.getByRole('heading', {
+        name: 'Gervimaður Evrópa',
+      })
+
+      const seeMoreText = 'Sjá nánar'
+      const seePaymentsButton = page.getByRole('link', {
+        name: seeMoreText,
+      })
+
+      // Assert
+      await expect(name).toBeVisible()
+
+      seePaymentsButton.click()
+
+      await page.waitForLoadState('networkidle')
+      expect(page.url()).toContain('#overview')
+    })
+  })
+
+  test('Medicine Payment page', async () => {
+    const page = await context.newPage()
+    await disableI18n(page)
+
+    await test.step('Should have medical step and invoices ', async () => {
+      // Arrange
+      await page.goto('/minarsidur/heilsa/lyf')
+
+      // Act
+      const medicalStep = page.getByText('Þrepastaða').first()
+
+      const invoices = page.getByTestId('invoices')
+
+      // Assert
+
+      await expect(medicalStep).toBeVisible()
+      await expect(invoices).toBeVisible()
+    })
+  })
+
+  test('Medicine Calculator page', async () => {
+    const page = await context.newPage()
+    await disableI18n(page)
+
+    await test.step(
+      'Should have drug filter and calculate button',
+      async () => {
+        // Arrange
+        await page.goto('/minarsidur/heilsa/lyf#reiknivel')
+
+        // Act
+        const filter = page.getByPlaceholder('Leita að lyf')
```

Comment:
Consider using i18n labels where possible (see @thordurhhh's comment above)
```suggestion
        const filter = page.getByPlaceholder(label('Leita að lyf'))
```